pkg/core/core.go

```diff
@@ -61,12 +61,17 @@ func SendDNSQuery(client *dns.Client, msg dns.Msg, dnsServerIP string) (*dns.Msg
 		}
 		dnsServerIP = conf.Servers[0]
 	}
-	logrus.Debugf("Sending DNS query to %s", dnsServerIP)
+
+	logrus.Debugf("Sending query to %s", dnsServerIP)
```

Comment:
Can you add 'DNS' before query, as we also log the word 'DNS' in front of reponse